### PR TITLE
fix(MintCollectibles): Collectible component updates

### PR DESCRIPTION
**Pull request by noeliaSD**
Closes [#10606](https://github.com/status-im/status-desktop/issues/10606)

### What does the PR do
- `CollectibleView`: Updated UI details according to new design
- Added available tokens and total supply as subtitle information in minted collectibles view
- Updated storybook model with new property `availableTokens`

### Affected areas

Wallet / Collectibles view
Community Settings / Collectibles view

### Screenshot of functionality 

- Mint tokens (storybook):

Screenshot 2023-05-12 at 11 49 00

- Wallet: 

https://github.com/status-im/status-desktop/assets/97019400/33a50059-75fb-4c9d-9456-a457d009d1ce

### CODE REVIEW

File: storybook/pages/CommunityCollectibleViewPage.qml [hunk 1]
```diff
@@ -24,7 +24,6 @@ SplitView {
 
                 anchors.fill: parent
                 anchors.margins: 50
-                holdersModel: TokenHoldersModel {}
```
Comment (caybro) [inline storybook/pages/CommunityCollectibleViewPage.qml:27]: Is this still needed then, can't we remove it?

**Review by caybro — APPROVED** [review]
LGTM!

File: storybook/pages/CommunityCollectibleViewPage.qml (hunk 1 above)
Comment (noeliaSD) [reply storybook/pages/CommunityCollectibleViewPage.qml:27]: It's not a property of `CommunityCollectibleView` but the model is already used to test other views!

**Review by dlipicar — APPROVED** [review]
looks great!

**Review by alexjba — APPROVED** [review]

**Pull request merged**